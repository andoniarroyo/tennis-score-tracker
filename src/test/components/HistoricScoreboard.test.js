import React from 'react';
import renderer from 'react-test-renderer';

import HistoricScoreBoard from '../../main/components/HistoricScoreboard';

describe('<HistoricScoreBoard />', () => {
  it('renders the historic scoreboard', () => {
    const props = {
      player1historicScoreboard: [6, 6],
      player2historicScoreboard: [4, 4],
    };

    const tree = renderer.create(<HistoricScoreBoard {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
