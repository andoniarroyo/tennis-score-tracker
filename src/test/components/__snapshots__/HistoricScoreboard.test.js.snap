// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HistoricScoreBoard /> renders the historic scoreboard 1`] = `
<section>
  <table>
    <thead>
      <tr>
        <td>
          <h3>
            Scoreboard:
          </h3>
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td
          className="player-historic-name"
        >
          player 1:
        </td>
        <td
          className="player-historic-value"
        >
          6
        </td>
        <td
          className="player-historic-value"
        >
          6
        </td>
      </tr>
      <tr>
        <td
          className="player-historic-name"
        >
          player 2:
        </td>
        <td
          className="player-historic-value"
        >
          4
        </td>
        <td
          className="player-historic-value"
        >
          4
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;
